### DISCUSSION TITLE
Cancel Villager Trading (Right-Click Event)

### DISCUSSION THREAD

**Discussion by Focxer** [EssentialsX/Essentials#4222] [Ideas and Feature Suggestions] [answered]
Would be great to cancel the Villager Trades on the server, i have a survival and i use another plugin to cancel the Right Click Event on Villagers

**Comment by DebugOk** [comment, marked as answer]
I'm not a maintainer of EssentialsX, but I really don't see why that should be a part of this plugin. Like you said, you have another plugin to do it for you, so why should Essentials need to do it?

**Reply by Focxer** [reply]
because it's something essential and would be awesome to have this in a single plugin like EssentialsX. The fewer plugins the better... you know im saying?

**Reply by ProblemsSender** [reply]
no i do not think that is a good idea, the plugin that already has cancel villager trading have much valuable option like modify what villager will trade etc, if you really just need to cancel that event. that plugin already gave you everything you need and without any performance impact whatsoever.
